### DISCUSSION TITLE
How to use zed's remote development function

### DISCUSSION THREAD

**Discussion by Jengro777** [zed-industries/zed#12272] [Feature Requests]
How to use zed's remote development function, no complete tutorial found for this

**Comment by SomeoneToIgnore** [comment]
https://zed.dev/docs/remote-development

> Remote Development is in the early stages of development. If you'd like to try it please email [alpha@zed.dev](mailto:alpha@zed.dev).

Do note though, that current remote development impl requires a direct connection to Zed's collab server both from the host and from the client, which is not suitable for certain remote development models.
